src/react/hooks/internal/useDeepMemo.ts

```diff
@@ -1,5 +1,5 @@
-import type { DependencyList } from "react";
-import * as React from "react";
+import type { DependencyList } from "rehackt";
```

Comment:
```suggestion
import type { DependencyList } from "react";
```

Should this import be from the `react` package?

Reply:
Tbh., I'd like some consistency here - we should probably use only one package here and not mix up two different ones, otherwise we'll end up with a mix of `react` and `rehackt` in the generated `.d.ts` files.